modules/mailer/mailer.go

```diff
@@ -32,8 +32,8 @@ func NewMessageFrom(to []string, from, subject, body string) *Message {
 	msg.SetHeader("To", to...)
 	msg.SetHeader("Subject", subject)
 	msg.SetDateHeader("Date", time.Now())
-	msg.SetBody("text/plain", body)
```

Comment:
I see. Have you tried a test email to confirm line break is now working as expected?